### PR TITLE
Update V15 example ProductCategoryConstant

### PR DESCRIPTION
**Pull request by danielfrg**
Renaming to ProductCategoryConstant for V15.

### CODE REVIEW

File: examples/shopping_ads/get_product_category_constant.rb [hunk 1]
```diff
@@ -63,17 +61,17 @@ def get_product_bidding_category_constant(customer_id)
   root_categories = Set.new
 
   response.each do |row|
-    product_bidding_category = row.product_bidding_category_constant
+    product_category = row.product_category_constant
     category = {
-      name: product_bidding_category.localized_name,
-      id: product_bidding_category.resource_name,
+      name: product_category.localizations,
```
Comment (jradcliff) [inline examples/shopping_ads/get_product_category_constant.rb:66]: The `localizations` field is a repeated field, so you need to find the `en`/`US` entry in that collection

Comment (danielfrg) [reply examples/shopping_ads/get_product_category_constant.rb]: Done

Comment (jradcliff) [reply examples/shopping_ads/get_product_category_constant.rb]: Would `select` followed by `first` be more idiomatic for Ruby?

Comment (danielfrg) [reply examples/shopping_ads/get_product_category_constant.rb]: I usually try to keep those a bit more generic but I changed it to `detect` which i think is more idiomatic.

Comment (jradcliff) [reply examples/shopping_ads/get_product_category_constant.rb]: Thanks! The updated version looks easier to read.

**Review by jradcliff — APPROVED** [review]

**Pull request merged**